engine-cockpit/src/ch/ivyteam/enginecockpit/system/administrators/AdministratorBean.java

```diff
@@ -63,23 +66,34 @@ public String getStepWarningMessage() {
   }
 
   public void createAdmin() {
-    AdministratorService.instance().save(admin.toAdministrator());
+    service.config().save(admin.toAdministrator());
     load();
     Message.info().summary("'" + admin.getName() + "' added").show();
   }
-  
+
   public void updateAdmin() {
-    AdministratorService.instance().save(admin.toAdministrator());
+    service.config().save(admin.toAdministrator());
```

Comment:
here you only do something on the config, while on the deleteAdmin function you also do stuff on the db. Not very clear to me when what is needed 🙈

Reply:
<img width="1581" height="658" alt="image" src="https://github.com/user-attachments/assets/8dbef429-6db3-4b7f-853e-b6b4ee3e889e" />